### PR TITLE
DOCSP-22847 New aggregation toolbar features

### PR DESCRIPTION
**Pull request by jocelyn-mendez1**
Updated Compass Aggregation Pipeline pages to include new export and explain plan feature along with the new run button. 

JIRA: https://jira.mongodb.org/browse/DOCSP-22847

STAGING: (Explain Plan) https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-22962/aggregation-pipeline-builder/#explain-plan-for-pipelines

(new toolbar and run button)
https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-22848/aggregation-pipeline-builder/ 

(Export Results): https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-22849/import-export/#export-data-from-a-collection

(Updating Screenshot for Export Pipeline to Specific Language): https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-22849/export-pipeline-to-language/#click-the-export-button

### CODE REVIEW

**Comment by jocelyn-mendez1** [comment]
Hi Max, I have added the new features run, export, and explain plan features for the aggregation toolbar. Let me know of any possible changes. Thanks!

**Comment by jocelyn-mendez1** [comment]
Hi @mmarcon, sorry about that I have update the PR to include the staging links!

File: source/import-export.txt [hunk 1]
```diff
@@ -158,6 +160,48 @@ Procedure
       .. include:: /includes/steps/export-filter.rst
 
       .. include:: /includes/fact-export-progress-bar.rst
+   
+   .. tab:: Export Aggregation Results
+      :tabid: export-Aggregation
+
+      To export results from your aggregation pipeline:
```
Comment (mmarcon) [inline source/import-export.txt:167]: Do you think it's worth having a reference to how to export the results of a pipeline also in the aggregation builder docs? Given that we are showing the `Export` button the UI we may want to say what it does.

File: source/includes/steps-create-agg-pipeline.yaml [hunk 2]
```diff
@@ -50,5 +50,22 @@ content: |
 
          .. figure:: /images/compass/agg-builder-exclude-stage-example.png
             :alt: Aggregation Builder exclude stage example
-...
+---
+title: Run Pipeline.
+level: 4
+ref: run-aggregation
+content: |
+  Click :guilabel:`Run` at the top right of the pipeline builder.
```
Comment (mmarcon) [inline source/includes/steps-create-agg-pipeline.yaml:58]: We may want to add an info/warning box to highlight the fact that when you hit `Run`, the aggregation runs against the entire collection.

**Review by mmarcon — COMMENTED** [review]
Looks good, thank you.

Left a couple of comments but other than that feel free to ship it!

**Pull request closed without merging**